Documentation/building/windows-instructions.md

```diff
@@ -19,8 +19,13 @@ We also require that [Visual Studio 2015 Update 1](https://www.visualstudio.com/
 ## Building From the Command Line
 
 Open a [Visual Studio Command Prompt](http://msdn.microsoft.com/en-us/library/ms229859(v=vs.110).aspx).
-From the root of the repository, type `build.cmd`. This will build everything and run
-the core tests for the project. 
+
+`build.cmd` - Will cause basic tool initialization and build the default configuration for refs, libs, and packages.
+`build-tests.cmd` - Will build and run tests for the default configuration.
+
+For information on different configurations [project-guidelines](../coding-guidelines/project-guidelines.md).
+
+*Note*: Before working on individual projects or test projects you **must** run `build.cmd` once before beginning that work.
 
```

Comment:
If I had to guess, the problem you're describing probably occurred when a project.json was changed and you didn't re-run the restore command for it. I don't think the same thing should necessarily happen in the new world, although there could be incompatible changes made in the ref build that necessitate a re-build for the src build.